src/test/java/org/fundacionjala/coding/danielcabero/movies/ChildrenTest.java

```diff
@@ -0,0 +1,80 @@
+package org.fundacionjala.coding.danielcabero.movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by administrator on 3/10/2017.
+ */
+public class ChildrenTest {
+    /**
+     * Test to verify the amount if the rented days
+     * for a Children movie is less than 3 e.g. 1
+     */
+    @Test
+    public void testCalculateAmountWhenTheRentedDaysIsLessThanThree() {
+        // given:
+        Children childrenInstance = new Children("Test");
+
+        // when:
+        final double actualResult = childrenInstance.calculateAmount(1);
+
+        // then
+        final double expectedResult = 1.5;
+        assertEquals(0, expectedResult, actualResult);
```

Comment:
first argument should be expectedResult and the second argument should be actualResult